layers/core_checks/image_validation.cpp

```diff
@@ -1661,6 +1661,55 @@ bool CoreChecks::PreCallValidateCreateImageView(VkDevice device, const VkImageVi
         image_usage = chained_ivuci_struct->usage;
     }
 
+    if (const auto sliced_create_info_ext = LvlFindInChain<VkImageViewSlicedCreateInfoEXT>(pCreateInfo->pNext);
+        sliced_create_info_ext) {
+        const bool feature_disabled = (enabled_features.sliced_3d_features.imageSlicedViewOf3D == VK_FALSE);
+        if (feature_disabled) {
+            skip |= LogError(pCreateInfo->image, "VUID-VkImageViewSlicedCreateInfoEXT-None-07871",
+                             "vkCreateImageView(): imageSlicedViewOf3D not supported.");
```

Comment:
nit: 

`supported` means the device support the feature
`enabled` means the user forgot to enable it

```suggestion
                             "vkCreateImageView(): imageSlicedViewOf3D not enabled.");
```